components/Proposals/ProposalDetailCard.tsx

```diff
@@ -145,15 +143,13 @@ export function ProposalDetailCard() {
         <Accordion w="100%" pb="md">
           <Accordion.Item value={proposal.publicKey.toString()}>
             <Accordion.Control>
-              <Stack>
```

Comment:
do we still need this `Stack` or is this a vestige?

Reply:
May be needed for something funky with the accordion..  But we'll allow it, if it breaks design, we know where to look.